### PR TITLE
[GHC-125] Bump hackage-search pin to fix queries with '/'

### PR DESCRIPTION
**Pull request by zhenyavinogradov**
Applies the fix from https://github.com/serokell/hackage-search/pull/19

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -9,7 +9,12 @@
       flake = false;
     };
     deploy-rs.url = "github:serokell/deploy-rs";
-    hackage-search.url = "github:serokell/hackage-search";
+    hackage-search = {
+      type = "github";
+      owner = "serokell";
+      repo = "hackage-search";
+      ref = "zhenya/ghc125-slash-issues";
```
Comment (int-index) [inline flake.nix:16]: The change is merged, so undo this?

Comment (zhenyavinogradov) [reply flake.nix:16]: I'll undo it before merging, I might have to rebase anyway

**Comment by int-index** [comment]
What’s the status here? Hackage Search is currently broken on `/`

**Review by mkaito — APPROVED** [review]

**Comment by zhenyavinogradov** [comment]
Deployed, the problem with `/` is fixed now

**Comment by int-index** [comment]
What about https://github.com/serokell/pegasus-infra/pull/11#discussion_r571996419?

**Comment by zhenyavinogradov** [comment]
Damn

**Pull request merged**